client/src/templates/Introduction/components/super-block-intro.tsx

```diff
@@ -25,17 +25,29 @@ function SuperBlockIntro(props: SuperBlockIntroProps): JSX.Element {
 
   return (
     <>
-      <h1 id='content-start' className='text-center big-heading'>
+      <h1
+        id='content-start'
+        className='text-center big-heading'
+        data-playwright-test-label='superblock-heading'
+      >
         {i18nSuperBlock}
       </h1>
       <Spacer size='medium' />
-      {generateIconComponent(superBlock, 'cert-header-icon')}
+      <div data-playwright-test-label='superblock-icon'>
+        {generateIconComponent(superBlock, 'cert-header-icon')}
+      </div>
       <Spacer size='medium' />
-      {superBlockIntroText.map((str, i) => (
-        <p key={i}>{str}</p>
-      ))}
+      <div data-playwright-test-label='superblock-description-box'>
+        {superBlockIntroText.map((str, i) => (
+          <p key={i}>{str}</p>
```

Comment:
@munzah-shah 

I prefer adding a data-playwright-label on paragraphs as well and use a consistent playwright method ( getByTestId ) unless necessary.